src/commands/org/login/device.ts

```diff
@@ -72,7 +71,7 @@ export default class LoginDevice extends SfCommand<DeviceLoginResult> {
     const loginData = await deviceOauthService.requestDeviceLogin();
 
     if (this.jsonEnabled()) {
-      ux.log(JSON.stringify(loginData, null, 2));
+      this.log(JSON.stringify(loginData, null, 2));
```

Comment:
I don't think `this.log` will actually log if JSON is enabled (plugins-core does that no-op thing)

Reply:
that makes sense - was wondering why `ux` was being used there